SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -0,0 +1,69 @@
+## Cross-server RPCs
+
+### Overview
+
+In Unreal’s native client-server architecture, your game server holds the canonical state of the whole game world. As there is a single game server, it has authority over all server Actors and so it is able to invoke and execute functions on Actors unhindered. 
+
+In SpatialOS games, there can be more than one server; these multiple servers are known as “server-workers”. (Find out more about server-workers as well as “client-workers” in the [glossary]({{URLRoot}}/content/glossary#workers).)  
+As a SpatialOS game runs across many server-workers, SpatialOS server-workers have the concept of “worker authority” - where only one server-worker at a time is able to invoke and execute functions on Actors. (Find out more about authority in the [glossary]({{URLRoot}}/content/glossary#authority).)
```

Comment:
The glossary links currently aren't present, @ElleEss could you confirm that these are valid please?

Reply:
I'll check all glossary links against the latest glossary GDoc.

Reply:
Thank you <3 - assuming I don't need to check as Anne's all over it?

Reply:
ALso, note that linting will catch this before we go live.